internal/boxcli/search.go

```diff
@@ -100,7 +102,12 @@ func printSearchResults(
 
 		versionString := ""
 		if len(nonEmptyVersions) > 0 {
-			versionString = fmt.Sprintf(" (%s)", strings.Join(nonEmptyVersions, ", "))
+			versionString = fmt.Sprintf(" (%s", strings.Join(nonEmptyVersions, ", "))
+		}
+		if resultsAreTrimmed && pkg.NumVersions > TrimmedVersionsLength {
+			versionString += " ...)"
+		} else {
+			versionString += ")"
```

Comment:
Suggestion to avoid closing paren manually:

```golang
ellipses := lo.Ternary(resultsAreTrimmed && pkg.NumVersions > TrimmedVersionsLength, " ...", "")
versionString = fmt.Sprintf(" (%s%s)", strings.Join(nonEmptyVersions, ", "), ellipses)
```